app-saml/config/authsources.php

```diff
@@ -0,0 +1,396 @@
+<?php
```

Comment:
I might be looking in the wrong place but I can't seem to find the comments you're referencing here in the pillar repo. is it on the saml branch?